LPD 27648

ealonso opened the pull request:
- LPD-27648 Creates new interface to represent Related Info Items
- LPD-27648 Return a list of RelatedInfoItems instead of a className
- LPD-27648 Implements new method
- LPD-27648 Return also the name
- LPD-27648 Adapt code and return the name
- LPD-27648 Update tests to reflect new information
- LPD-27648 baseline

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

victorg1991 commented:
ci:test:sf

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-18.liferay.com/job/test-portal-source-format/1430/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-18.liferay.com//userContent/jobs/test-portal-source-format/builds/1430/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 5 minutes 6 seconds 881 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/819479b2c9e598395f8b35321ef74a9481cb24d5">819479b2c9e598395f8b35321ef74a9481cb24d5</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/victorg1991/liferay-portal/tree/LPS-97538">LPS-97538</a><br/>Branch GIT ID: <a href="https://github.com/victorg1991/liferay-portal/commit/ffce120d69a71366b125dcb169e632726741df10">ffce120d69a71366b125dcb169e632726741df10</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-18.liferay.com/job/test-portal-source-format/1430/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-18.liferay.com/userContent/jobs/test-portal-source-format/builds/1430/jenkins-report.html">here</a>.</h5></details></html>

sandrodw3 commented:
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

ealonso committed:
LPD-27648 Creates new interface to represent Related Info Items

ealonso committed:
LPD-27648 Return a list of RelatedInfoItems instead of a className

ealonso committed:
LPD-27648 Implements new method

ealonso committed:
LPD-27648 Return also the name

ealonso committed:
LPD-27648 baseline

# breaking

## What modules/apps/info/info-api/src/main/java/com/liferay/info/item/provider/RelatedInfoItemProvider.java

The following method:

public List<String> getRelatedItemClassNames()

was replaced by:

public List<RelatedInfoItem> getRelatedInfoItems()

## Why

This change is necessary to manage relationships in content page editor

----

ealonso committed:
LPD-27648 Adapt code and return the name

ealonso committed:
LPD-27648 Update tests to reflect new information

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.
